types.d.ts

```diff
@@ -55,6 +55,11 @@ export interface BrowserStatement {
    */
   type: BrowserTypes;
 
+  /**
+   * The upstream browser
+   */
+  upstream?: string;
```

Comment:
Oh, this makes me think that that the `meta: { version }` is not yet declared here. 🤔